webhooks-extension/README.md

```diff
@@ -1,18 +1,72 @@
+webhooks-extension/README.md
+
 # Webhooks Extension
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubernetes/experimental/blob/master/LICENSE)
 
-The Webhooks Extension for Tekton allows users to set up Git webhooks that will trigger Tekton PipelineRuns and TaskRuns. An initial implementation will use Knative Eventing but we're closely following the eventing discussion in  [Tekton Pipeline](https://github.com/tektoncd/pipeline) to minimize necessary componentry.
+The Webhooks Extension for Tekton allows users to set up GitHub webhooks that will trigger Tekton `PipelineRuns` and associated `TaskRuns`. This initial implementation utilises Knative Eventing but we're closely following the eventing discussion in  [Tekton Pipeline](https://github.com/tektoncd/pipeline) to minimize necessary componentry.
 
 In addition to Tekton/Knative Eventing glue, it includes an extension to the Tekton Dashboard.
 
-## Runtime Dependencies
-- [Tekton](https://github.com/tektoncd/pipeline) & [Tekton Dashboard](https://github.com/tektoncd/dashboard)
-- Knative [eventing](https://knative.dev/docs/eventing/), [eventing sources](https://knative.dev/docs/eventing/sources/), & [serving](https://knative.dev/docs/serving/)
+## Prerequisites
+- Install [Tekton](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) _(version 0.3.0 is recommended)_ and the [Tekton Dashboard](https://github.com/tektoncd/dashboard)
```

Comment:
Version 0.4 was released 29 days ago - it might be simpler to remove a version recommendation.